phase-0-tracks/ruby/vampires.rb

```diff
@@ -0,0 +1,110 @@
+puts "How many employees will be processed?"
+
+employees = gets.to_i
+
+i = 0
+
+until i > employees.to_i
+
+puts "What is your Name?"
+
+e_name = gets.chomp
+
+valid_input = FALSE
+until valid_input 
+
+puts "How old are you? What year were you born?"
+
+age = gets.to_i
+year = gets.to_i
+
+if age.to_i != (2017-year)
+  valid_input = TRUE
+  puts "MisMatch"
+elsif 
+  age.to_i = (2017-year)
+  valid_input = TRUE
+  puts "Match"
+else
+  puts "Please enter your age and number"
+end
+end
+
+bread_input = FALSE
+until bread_input
+
+puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
+
+bread_yn = gets.chomp
+
+if bread_yn == "y"
+bread_input = TRUE
+puts "yes"
+elsif bread_yn == "n"
+bread_input = TRUE
+puts "no"
+else
+  puts "please reply y or n"
+end
+end
+
+I_input = FALSE
+until I_input 
+
+puts "Would you like to enroll in the company’s health insurance?(y/n)"
+
+i_yn = gets.chomp
+
+if i_yn == "y"
+I_input = TRUE
+puts "yes"
+elsif i_yn == "n"
+I_input = TRUE
+puts "no"
+else
+  puts "reply with y or n"
+  
+end
+end
+
+if e_name == ("Drake Cula") || ("Tu Fang") 

```

Comment:
Consider nesting your control flow.  In this case if the we know the employee's name is Drac of Fang then we don't need to run anymore of these conditions. 

```
if name == "Drake Cula" || name == "Tu Fang"
  results = "Definitely a vampire"
else
  # run other checks
end
```